src/app/page.tsx

```diff
@@ -33,31 +20,15 @@ interface DataEntry {
 	revocationTime: number;
 }
 
-function sortByStartsAt(data: DataEntry[]): DataEntry[] {
-	return data.sort((a, b) => {
-		const aDecoded = JSON.parse(a.decodedDataJson);
-		const bDecoded = JSON.parse(b.decodedDataJson);
-
-		const aStartsAt = aDecoded.find((item: any) => item.name === "startsAt")
-			?.value.value.hex;
-		const bStartsAt = bDecoded.find((item: any) => item.name === "startsAt")
-			?.value.value.hex;
-
-		if (!aStartsAt || !bStartsAt) return 0;
-
-		return parseInt(aStartsAt, 16) - parseInt(bStartsAt, 16);
-	});
-}
-
 export default function Events() {
 	const signer = useSigner();
 	const chainId = useChainId();
 	const attester = signer?.address || "";
 	const [searchTerm, setSearchTerm] = useState("");
 
 	const [result] = useQuery({
-		query: USER_ATTESTATIONS_QUERY,
-		variables: { attester },
+		query: EVENTS_ATTESTATIONS_QUERY,
+		variables: { schemaId: EVENT_SCHEMA_ID },
 		context: useMemo(() => ({ url: API_URL_MAPPING[chainId] }), [chainId]),
 		pause: !signer,
 	});
```

Comment:
`#would` create a useAttestations hook and use it in both files `src/app/page.tsx` and `src/app/events/page.tsx`

Something like that:

export const useAttestations = (signer: any) => {
  const chainId = useChainId();
  const attester = signer?.address || "";

  const [result] = useQuery({
    query: USER_ATTESTATIONS_QUERY,
    variables: { attester, schemaId: EVENT_SCHEMA_ID },
    context: useMemo(() => ({ url: API_URL_MAPPING[chainId] }), [chainId]),
    pause: !signer,
  });

  return result;
};

Reply:
sure! this PR was originally from Gabriel, didn't want to change his code too much xD